developer-docs/bolt_server.md

```diff
@@ -0,0 +1,39 @@
+# Bolt Server 
+
+## Overview
+Bolt server provides a service to run bolt tasks over SSH and WinRM. This service is exposed via an API as described in this document. 
+
+## Configuration
+Bolt server can be configured by defining content in HOCON format at one of the following expected configuration file path locations.
+**Global Config**: ` /etc/puppetlabs/bolt-server/conf.d/bolt-server.conf`
+**Local Config**: `~/.puppetlabs/bolt-server.conf`
+**Options**
+`port [optional int]`: The port the bolt server will run on (default 8144)
+`ssl-cert [required string]`: Path to the cert file
+`ssl-key [required string]`: Path to the key file
+`ssl-ca-cert [required string]`: Path to the ca cert file
+
+**Example**
+```
+bolt-server: {
+    port: 8144
+    ssl-cert: cert.pem
```

Comment:
Might make these absolute paths? In PE they'll default to `/etc/puppetlabs/bolt-server/ssl/*.pem`